package/network/services/lldpd/files/lldpd.init

```diff
@@ -188,6 +198,8 @@ write_lldpd_conf()
 	[ -n "$lldp_platform" ] && echo "configure system platform" "\"$lldp_platform\"" >> "$LLDPD_CONF"
 	[ -n "$lldp_tx_interval" ] && echo "configure lldp tx-interval $lldp_tx_interval" >> "$LLDPD_CONF"
 	[ "$lldp_tx_hold" -gt 0 ] && echo "configure lldp tx-hold $lldp_tx_hold" >> "$LLDPD_CONF"
+	[ "$lldp_capadv" -gt 0 ] && echo "configure lldp capabilities-advertisements" >> "$LLDPD_CONF"
```

Comment:
I think as a bool that `-ne 0` would be more readable/intuitive.

Reply:
Valid. As long as both work with potentially non-boolean values. Some settings (elsewhere) have been co-opted to later become ternary or greater choice settings.

Reply:
Here's the thing: the rest of the file uses `-gt 0` so it seems like a refactor is in order, just so these two `-ne 0` don't raise questions later of "why did they do that?"

Reply:
Sorry, I thought the others were for integers and not booleans.

In any case, not crucial.